source/resources/index.haml

```diff
@@ -3,43 +3,37 @@ title: Resources
 layout: 'layout'
 ---
 - content_for :header do
-  -# NOTE: Add .aptible-header--with-breadcrumbs when restoring filters
-  %header.aptible-header.hero-gradient--angled
-    = partial 'partials/main-nav', locals: {      |
-        section_title: 'Resources',               |
-        section_url: '/resources/'                |
-      }                                           |
-
--# TODO: Add filtering when we have more resources
--# .filter-state
--#   .grid-container
--#     %span.filter-state__subject Resources
--#     %span.filter-state__tag All
+  - if featured_resource
+    = partial 'partials/headers/featured_resource'
+  - else
+    %header.aptible-header.hero-gradient--angled
+      = partial 'partials/main-nav'
```

Comment:
If there are no featured resources from contentful, we get a standard header: 
<img width="1325" alt="screen shot 2017-07-21 at 2 39 28 pm" src="https://user-images.githubusercontent.com/94830/28481496-7075128e-6e22-11e7-81a7-858621e21be0.png">